### PR TITLE
Changes before the new `Time` module.

### PR DESCRIPTION
**Pull request by gdotdesign**

### CODE REVIEW

File: core/tests/tests/String.mint [hunk 1]
```diff
@@ -222,3 +222,15 @@ suite "String.indentWithOptions" {
...
 }
+
+suite "String.padLeft" {
+  test "it pads the string from the left" {
+    String.padLeft("0", 2, "5") == "05"
+  }
+}
+
+suite "String.takeRight" {
+  test "it pads the string from the left" {
```
Comment (Sija) [inline core/tests/tests/String.mint:233]: Copy-pasted spec name.

**Review by Sija — APPROVED** [review]
LGTM

File: core/tests/tests/Math.mint [hunk 2]
```diff
@@ -80,6 +80,12 @@ suite "Math.fmod" {
   }
 }
 
+suite "Math.truncate" {
+  test "it truncates number to given amount" {
```
Comment (Sija) [inline core/tests/tests/Math.mint:84]: Copy pasted from `Math.truncate`.

Comment (gdotdesign) [reply core/tests/tests/Math.mint:84]: I don't understand your comment :) please elaborate.

Comment (Sija) [reply core/tests/tests/Math.mint:84]: It's copy pasted from `Math.truncate` specs - which is a different method.

Comment (gdotdesign) [reply core/tests/tests/Math.mint:84]: Ah, I see now the name should be `Math.trunc`.

**Pull request merged**